nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/cmd/exec/internal/search/DefaultSearchFunctionHandlerLookup.java

```diff
@@ -93,24 +100,57 @@ private R getStaticParamValues(List<StaticCodeValue> searchResult, Command cmd)
 		return (R) searchResult.get(0).getParamValues();
 	}
 	
+//	private R getDynamicParamValues(LookupSearchCriteria lookupSearchCriteria, Class<?> criteriaClass, List<?> searchResult) {
+//		List<String> list = new ArrayList<String>(lookupSearchCriteria.getProjectCriteria().getMapsTo().values());
+//		
+//		if(list.size() > 2)
+//			throw new IllegalStateException("ParamValues lookup failed due to more than 2 fields provided to create the param values. the criteria class is "+criteriaClass);
+//		
+//		PropertyDescriptor codePd = CustomBeanUtils.getPropertyDescriptor(criteriaClass, list.get(0));
+//		PropertyDescriptor labelPd = CustomBeanUtils.getPropertyDescriptor(criteriaClass, list.get(1));
+//		String[] names = list.get(1).split("\\.");
+//		try {
+//			List<ParamValue> paramValues = new ArrayList<>();
+//			Object mod;
+//			for(Object model: searchResult) {
+//				if(names.length > 1) {
+//					mod = PropertyUtils.getProperty(model, names[names.length - 2]);
+//				} else {
+//					mod = model;
+//				}
+//				paramValues.add(new ParamValue(codePd.getReadMethod().invoke(model).toString(),labelPd.getReadMethod().invoke(mod).toString()));
+//			}
+//			return (R)paramValues;
+//		}
+//		catch(Exception ex) {
+//			throw new FrameworkRuntimeException("Failed to execute read on property: "+codePd+" and "+labelPd, ex);
+//		}
+//	}
+	
 	private R getDynamicParamValues(LookupSearchCriteria lookupSearchCriteria, Class<?> criteriaClass, List<?> searchResult) {
 		List<String> list = new ArrayList<String>(lookupSearchCriteria.getProjectCriteria().getMapsTo().values());
-		
+
 		if(list.size() > 2)
-			throw new IllegalStateException("ParamValues lookup failed due to more than 2 fields provided to create the param values. the criteria class is "+criteriaClass);
-		
-		PropertyDescriptor codePd = BeanUtils.getPropertyDescriptor(criteriaClass, list.get(0));
-		PropertyDescriptor labelPd = BeanUtils.getPropertyDescriptor(criteriaClass, list.get(1));
+		throw new IllegalStateException("ParamValues lookup failed due to more than 2 fields provided to create the param values. the criteria class is "+criteriaClass);
 		
+		String cd = list.get(0).replaceAll("\\.", "\\?.");
+		String lb = list.get(1).replaceAll("\\.", "\\?.");
```

Comment:
Duplicated constants `"\\."` and `"\\?."`. Consider externalizing or adding these as a static constants.